robottelo/logging.py

```diff
@@ -0,0 +1,89 @@
+import logging
+import os
+from pathlib import Path
+
+import logzero
+import yaml
+from attrdict import AttrDict
+
+
+robottelo_root_dir = Path(os.environ.get('ROBOTTELO_DIR', Path(__file__).resolve().parent.parent))
+robottelo_log_dir = robottelo_root_dir.joinpath('logs')
+robottelo_log_file = robottelo_log_dir.joinpath('robottelo.log')
+robottelo_log_file.parent.mkdir(parents=True, exist_ok=True)
+
+with robottelo_root_dir.joinpath('logging.yaml').open() as f:
+    logging_yaml = AttrDict(yaml.load(f, yaml.FullLoader))
+
+DEFAULT_DATE_FORMAT = '%Y-%m-%d %H:%M:%S'
+
+defaultFormatter = logzero.LogFormatter(
+    fmt='%(asctime)s - %(name)s - %(levelname)s - %(message)s', datefmt=DEFAULT_DATE_FORMAT
+)
+
+logger = logzero.setup_logger(
```

Comment:
I ran into a lot of issues trying to use the logzero root logger for robottelo, and instead opted to create a specific logger.